Gems/Atom/Feature/Common/Assets/Materials/Types/NewPBR/NewPBR_LightingData.azsli

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <viewsrg.srgi>
+#include <Atom/Features/Debug.azsli>
+#include <Atom/Features/LightCulling/LightCullingTileIterator.azsli>
+#include <Atom/Features/PBR/LightingUtils.azsli>
+
+class LightingData
+{
+    LightCullingTileIterator tileIterator;
+    
+    // Lighting outputs
+    float3 diffuseLighting;
+    float3 specularLighting;
+
+    // Factors for the amount of diffuse and specular lighting applied
+    float3 diffuseResponse;
+   
+    // Shrink (absolute) offset towards the normal opposite direction to ensure correct shadow map projection
+    float shrinkFactor;
+
+    // Attenuation applied to hide artifacts due to low-res shadow maps
+    float distanceAttenuation;
```

Comment:
Small thing: `shrinkFactor` and `distanceAttenuation` are redundant and were removed in https://github.com/o3de/o3de/pull/8352, but somehow these came back at some point. Could you remove them?